src/main/java/org/zeromq/ZMQException.java

```diff
@@ -4,17 +4,8 @@
 
 public class ZMQException extends RuntimeException
 {
-    public static class IOException extends RuntimeException
-    {
-        private static final long serialVersionUID = 8440355423370109164L;
-
-        public IOException(java.io.IOException cause)
-        {
-            super(cause);
-        }
-    }
 
-    private static final long serialVersionUID = 5957233088499712341L;
+    private static final long serialVersionUID = -978820750094924644L;
```

Comment:
Ah, OK. Makes sense to me. It does seem like to keep JeroMQ consistent with jzmq, they two exceptions (which ought to be equivalent) should have the same serializable version ID.

Thanks for the explanation!